### PR TITLE
Add inline source maps to JavaScript files

### LINKED ISSUE #10: JavaScript source maps missing in development

### PR DESCRIPTION
**Pull request by incon**
Resolves #10

### CODE REVIEW

File: webpack.mix.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ const tailwindcss = require('tailwindcss')
  |
  */
 
+// Provide inline source maps.
+if ( ! mix.inProduction()) {
+  mix.webpackConfig({
+    devtool: 'inline-source-map',
```
Comment (reinink) [inline webpack.mix.js]: Curious, why is this needed? Is `sourceMaps()` not enough?

Comment (incon) [reply webpack.mix.js]: I'm afraid not, it's an known issue.
https://github.com/JeffreyWay/laravel-mix/issues/1793

Comment (incon) [reply webpack.mix.js]: Also I didn't use `devtool: 'source-map'`. You likely don't want the extra files ending up in `/public` and getting accidentally committed.

Comment (reinink) [reply webpack.mix.js]: Are you sure? I just tried with only `.sourceMaps()`, and it seemed to work fine. Or am I missing something?

image
